datamodel/src/main/kotlin/com/hartwig/actin/datamodel/algo/TreatmentMatch.kt

```diff
@@ -10,6 +11,22 @@ data class TreatmentMatch(
     val trialMatches: List<TrialMatch>,
     val standardOfCareMatches: List<AnnotatedTreatmentMatch>?,
     val personalizedDataAnalysis: PersonalizedDataAnalysis?,
-    val survivalPredictionsPerTreatment: Map<String, List<Double>>?,
+    val survivalPredictionsPerTreatment: Map<String, TreatmentEfficacyPrediction>?,
```

Comment:
I like the name of the class `TreatmentEfficacyPrediction`! Would also update the variable though (`efficacyPredictionPerTreatment`)